src/mapport.cpp

```diff
@@ -47,7 +47,7 @@ static void ProcessPCP()
     // Local functor to handle result from PCP/NATPMP mapping.
     auto handle_mapping = [&](std::variant<MappingResult, MappingError> &res) -> void {
         if (MappingResult* mapping = std::get_if<MappingResult>(&res)) {
-            LogPrintLevel(BCLog::NET, BCLog::Level::Info, "portmap: Added mapping %s\n", mapping->ToString());
+            LogInfo("portmap: Added mapping %s", mapping->ToString());
```

Comment:
Here and elsewhere these messages had a category before and now they are category-less. When a message has a category its log is prefixed by e.g. `[net:info]` which makes it easier to filter/grep all "net" messages and to know which module a message comes from.

Would be nice to have the ability to log info/warning/error messages with category.